azure-pipelines.yml

```diff
@@ -20,7 +20,9 @@ trigger:
 jobs:
 - job: InitializePipeline
   pool:
-    vmImage: 'windows-2019'
+    name: '1ES-Hosted-AzFunc'
+    demands:
```

Comment:
in the .net worker, I am using `vmImage:` instead of this `demands:`.  How is `demands` different from that approach? Should we keep it consistent?

Reply:
Thanks! I will fix .net worker as well.